src/tabs/Address__c.tab

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomTab xmlns="http://soap.sforce.com/2006/04/metadata">
     <customObject>true</customObject>
-    <mobileReady>false</mobileReady>
+    
```

Comment:
Can we remove these blank rows from the metadata? (For all the tab files with similar changes)